Cargo.lock

```diff
@@ -587,24 +666,24 @@ dependencies = [
 
 [[package]]
 name = "openssl-probe"
-version = "0.1.4"
+version = "0.1.5"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "28988d872ab76095a6e6ac88d99b54fd267702734fd7ffe610ca27f533ddb95a"
+checksum = "ff011a302c396a5197692431fc1948019154afc178baf7d8e37367442a4601cf"
 
 [[package]]
 name = "openssl-src"
-version = "300.0.2+3.0.0"
+version = "111.17.0+1.1.1m"
```

Comment:
Did cargo revert the version from OpenSSL 3.0 to 1.1.1??

Reply:
Hmmm good catch, let me do some digging.

Reply:
Looks like it wasn't Cargo but the openssl crate intentionally downgraded:

https://github.com/sfackler/rust-openssl/pull/1578

I don't think we really care which version is used, but I will verify that this works and post an update here.

Reply:
Yeah, no issue with openssl 1.1.1, it works the same to us.

Long term goal is to get rid of openssl entirely, but for now we should be good.